caf_solution/variables.maps.tf

```diff
@@ -0,0 +1,7 @@
+variable "maps" {
+ default = {}
+}
+
+variable "maps_account" {
```

Comment:
```suggestion
variable "maps_accounts" {
```

Reply:
Hi @arnaudlh, I have just updated. Please help to review it again.